### DISCUSSION TITLE
What's the data limits for GitHub Education?

### DISCUSSION THREAD

**Discussion by softwarelab3** [community/community#52036] [GitHub Education]
as above

**Comment by matej-kaska** [comment]
For GitHub Education you will get:
Unlimited public/private repos
Unlimited collaborators
3,000 Actions minutes/month
2GB of Packages storage
180 core-hours of Codespaces compute
20GB of Codespaces storage
Code owners, required reviewers, GitHub Pages, and more [advanced tools](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/githubs-products#github-team).

Feel free to accept my answer. :)

**Reply by softwarelab3** [reply]
HI, thanks.
By unlimited private repo, does it include all the education features or only limited?
2GB storage per repo or for one education license?
I'm in an institution where computer science student are uploading tons of data. If I create multiple classrooms, how much storage would I get each?

**Reply by matej-kaska** [reply]
Unlimited private repos mean that on your account with GitHub Education, you can get unlimited number of repos. Those repos can have I think unlimited size, but after 2GB or 5GB you will receive email from GitHub, that will say that they recommend you to have your repo smaller. So for repos there is not hard limit in documentation.

Feel free to accept my answer. :)

**Comment by samus-aran** [comment]
Hi @softwarelab3 

We offer **Git Large File Storage bundles**. Check out this [documentation on its pricing](https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage).

**Reply by davisjm4** [reply]
From what I understand, LFS is free up to 1gb with personal/free org accounts, is there an upgraded amount for education or the campus program?
